MapboxCoreNavigation/NavigationLocationManager.swift

```diff
@@ -23,7 +23,7 @@ open class NavigationLocationManager: CLLocationManager {
         }
         
         if #available(iOS 9.0, *) {
-            if Bundle.main.backgroundModeLocationSupported {
+            if Bundle.main.backgroundModes.location {
```

Comment:
Then this line would become `Bundle.main.backgroundModes.contains(.location)` or `Bundle.main.backgroundModes.contains("location")`.